recipe/meta.yaml

```diff
@@ -1,123 +1,206 @@
-{% set name = "cuda-python" %}
-{% set version = "12.6.2" %}
+{% set version = "0.1.1" %}
 {% set major_version = version.split(".")[0]|int %}
 {% set minor_version = version.split(".")[1]|int %}
 {% set patch_version = version.split(".")[2]|int %}
+{% set pkg_ver = major_version ~ "." ~ minor_version ~ "." ~ patch_version %}
+{% if major_version != 0 %}
+  {% set building_bindings = true %}
+  {% set building_core = false %}
+  {% set tag = "v" ~ pkg_ver %}
+  {% set number = 2 %}
+{% else %}
+  {% set building_bindings = false %}
+  {% set building_core = true %}
+  {% set tag = "cuda-core-v" ~ pkg_ver %}
+  {% set number = 0 %}
+{% endif %}
```

Comment:
With this PR, this feedstock starts to produce multiple outputs. In the future it'd be 
- `cuda-core`
- `cuda-bindings`
- `cuda-python`

as described in the upstream issue (https://github.com/NVIDIA/cuda-python/issues/105). For this particular tag, however, we only release `cuda-core` and the bindings should be kept intact, so a deselection logic is implemented here to only build `cuda-core`. We'll need to figure out how the logic looks like depending on the situation of future releases. Right now we're still experimenting so this is OK.